sdk/freestanding-execution-engine/README.md

```diff
@@ -0,0 +1,36 @@
+# Goal of the exercise
+
+The purpose of this exercise is to get familiarised with how to load files from the host and into the Veracruz freestanding execution engine. The freestanding execution engine inside of the Veracruz instance allows a user's program to be executed.
+
+IMPORTANT: This exercise assume some familiarity with reading **Rust syntax** and **Rust's package manager named Cargo**. Both these areas are well explained at this [link](https://doc.rust-lang.org/book/title-page.html).
+
+# Steps:
+1. Install [Rust](https://www.rust-lang.org/tools/install) and [wasmtime](https://wasmtime.dev/).
+2. Clone the Veracruz repository from [Github](https://github.com/veracruz-project/veracruz)
+3. Open the project in VS code for ease of visualising the project file structure
+4. Open up a terminal and go to the directory `veracruz/sdk/rust-examples`. In this example we will be focusing on the read-file directory
+5. Run the following commands
+  ```
+  rustup target add wasm32-wasi
+  cargo build --target wasm32-wasi
+  ```
+  The first command adds Web assembly (WASM) as the compilation target, so the rust program can be compiled to WASM. The second command complies the code that is     inside src, which is usually a file named main.rs, into WASM. Once the second command has been run you should expect to see a WASM file inside the directory `read-   file/target/wasm32-wasi/debug/read-file.wasm`.
```

Comment:
sp: Web assembly --> WebAssembly

There's also a tab character between "is" and "inside" (and between `read-` and `file`).

Put backticks around `main.rs`?